MotionDetection.py

```diff
@@ -24,6 +24,14 @@ def __init__(self, display_detections = True):
 		self._current_frame = None
 		self._current_gray_frame = None
 		self._display_detections = display_detections
+		self._default_img = cv2.imread("lib/default_img.png")
```

Comment:
is this a default image included in cv2 or do we need to push a lib folder with this image?

Reply:
This is an image that we would include in the "lib" folder.
If this is something you want to get away from, we can just create a basic "default_img" using some of the opencv GUI tools.

Reply:
I did include both default_img.png and the lib folder in this commit